mgplvm/lpriors/euclidean.py

```diff
@@ -108,7 +108,7 @@ def forward(self, x, batch_idxs=None):
     @property
     def msg(self):
         ell = self.svgp.kernel.prms[1].mean()
-        noise = self.svgp.likelihood.prms.sqrt()
+        noise = self.svgp.likelihood.prms.sqrt().mean()
```

Comment:
should we also add the following property:

```
@property
def sigma(self):
      return (1e-20 + self.prms).sqrt()
```

and use that here?

Reply:
probably a good idea